components/nexusoperations/config.go

```diff
@@ -68,6 +70,32 @@ ScheduleNexusOperation commands with an operation name that exceeds this limit w
 Uses Go's len() function to determine the length.`,
 )
 
+var MaxOperationHeaderSize = dynamicconfig.NewNamespaceIntSetting(
+	"component.nexusoperations.limit.header.size",
+	4096,
+	`The maximum allowed header size for a Nexus Operation.
+ScheduleNexusOperation commands with a "nexus_header" field that exceeds this limit will be rejected.
+Uses Go's len() function on header keys and values to determine the total size.`,
+)
+
+var DisallowedOperationHeaders = dynamicconfig.NewNamespaceTypedSettingWithConverter(
+	"component.nexusoperations.disallowedHeaders",
+	func(a any) ([]string, error) {
+		keys, ok := a.([]string)
+		if !ok {
+			return nil, fmt.Errorf("expected a string slice, got: %v", a)
+		}
+		for i, k := range keys {
+			keys[i] = strings.ToLower(k)
```

Comment:
These are intended to be HTTP headers, right? Might make more sense to use `http.CanonicalHeaderKey` to normalize instead of just lower casing.

Reply:
Headers are case insensitive so it really doesn't matter either way.